src/USD.NET.Unity/UnityTypeConverter.cs

```diff
@@ -28,23 +35,18 @@ public class UnityTypeConverter : IntrinsicTypeConverter {
     /// </summary>
     static public UnityEngine.Matrix4x4 ChangeBasis(UnityEngine.Matrix4x4 input) {
       // TODO(jcowles): the change of basis matrix should probably be cached.
-      var basisChange = UnityEngine.Matrix4x4.identity;
-      // Invert the forward vector.
-      basisChange[2, 2] = -1;
 
       // Note that the fully general solution is basisChange * m * basisChange.inverse, however
       // basisChange and basisChange.inverse are identical. Just aliasing here so the math below
       // reads correctly.
-      var basisChangeInverse = basisChange;
+      var basisChangeInverse = UnityTypeConverter.basisChange;
```

Comment:
It is no longer guaranteed that the inverse is the same as the basisChange now -- not sure if you don't care about that (just leave it as-is) or want to expose basisChangeInverse as well...

Reply:
I computed the inverse of the matrix with -1 in [0, 0] and it gave me the same matrix.

Reply:
Yes, for your example they are the same, but now that matrix can be set to anything, there is no guarantee that the it is involutory, or even invertable at all.

Reply:
Right... but do we really allow any matrix?... it's true that I don't do any verification :/
Should I add a check or at least a comment/warning above?

Reply:
Well, we don't support any matrix, but we allow it because the matrix is public.

You could just expose the inverse matrix also and leave it to the user to set both correctly. That would also have the nice side effect of caching the inverse matrix, instead of computing it every time.

BTW, this matrix probably should be initialized to the default value, otherwise it only has the correct default when calling APIs via SceneImporter.